src/Makefile.am

```diff
@@ -572,16 +572,16 @@ nodist_libbitcoin_util_a_SOURCES = $(srcdir)/obj/build.h
 #
 
 # bitcoind & bitcoin-node binaries #
```

Comment:
Update comment?

Reply:
The changes present in Makefile.am are purely functional; there is no point changing something that does not affect the build directly, that could create a future merge issue, no matter how insignificant.